src/main/scala/it/pagopa/interop/authorizationmanagement/model/persistence/projection/ClientCqrsProjection.scala

```diff
@@ -26,59 +26,62 @@ object ClientCqrsProjection {
     CqrsProjection[Event](offsetDbConfig, mongoDbConfig, projectionId, eventHandler)
 
   private def eventHandler(collection: MongoCollection[Document], event: Event): PartialMongoAction = event match {
```

Comment:
I don't see any benefit in readability in this case.
In fact, it makes commands longer and (in other services) less readable